rasa_core/events/__init__.py

```diff
@@ -230,17 +230,21 @@ def _from_story_string(cls, parameters):
         except KeyError as e:
             raise ValueError("Failed to parse bot uttered event. {}".format(e))
 
-    def as_story_string(self):
+    def as_story_string(self, e2e=False):
         if self.intent:
             if self.entities:
                 ent_string = json.dumps({ent['entity']: ent['value']
                                          for ent in self.entities})
             else:
                 ent_string = ""
 
-            return "{intent}{entities}".format(
-                    intent=self.intent.get("name", ""),
-                    entities=ent_string)
+            parse_string = "{intent}{entities}".format(
+                           intent=self.intent.get("name", ""),
+                           entities=ent_string)
+            if e2e:
+                return "{}:{}".format(parse_string, self.text)
```

Comment:
The end-to-end format we've been using so far uses md-style messages that include entities:
`* inform: on the [south](location) part of town`

Can we use `rasa_core.evaluate._md_format_message()` and maybe move it here?